packages/calcite-components/src/components/graph/graph.scss

```diff
@@ -4,12 +4,12 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-graph-background-color: Specifies the background color of the component.
- * @prop --calcite-graph-highlight-color: Specifies the highlight color of the component.
+ * @prop --calcite-graph-accent-color: Specifies the accent color of the component.
  */
 
 :host {
   --calcite-graph-background-color: currentColor;
-  --calcite-graph-highlight-color: var(--calcite-color-brand);
+  --calcite-graph-accent-color: var(--calcite-color-brand);
```

Comment:
Can you refactor this to match the new fallback component pattern?

Reply:
Thanks for the catch!